akka-http-core/src/main/resources/reference.conf

```diff
@@ -52,6 +52,12 @@ akka.http {
     # The time period within which the TCP binding process must be completed.
     bind-timeout = 1s
 
+    # Default value for the http port on which the server will be running.
+    default-http-port = 80
+
+    # Default value for the https port on which the server will be running.
```

Comment:
"Default port to bind HTTPS server to when no port was explicit given."

Reply:
same as above.

Reply:
Same as above.